Docs/userguide/debugging_guidelines.rst

```diff
@@ -4,35 +4,30 @@
 Quantization debugging guidelines
 #################################
 
-AIMET supports various neural network quantization techniques. If the model's performance is
-still not satisfactory after applying an AIMET quantization feature, we recommend a set of
-diagnostic steps to identify the bottlenecks and improve performance. These debugging steps can
-provide insights as to why a quantized model under performs and help to address the underlying
-issues, but they are not algorithmic.
+AIMET supports various neural network quantization techniques. If the model's performance is not satisfactory after applying an AIMET quantization feature, we recommend a set of diagnostic steps to identify the bottlenecks and improve performance. These debugging steps can point to why a quantized model underperforms and help to address the underlying issues, but they are not algorithmic.
 
 Debugging workflow
 ==================
 
-The steps are shown as a flow chart in the following figure and are described in more detail below:
+The steps are shown as a flow chart in the following figure and are described in more detail below.
 
 .. image:: ../images/debugging_guidelines_1.png
-   :height: 500
+   :height: 800
 
-
-1. FP32 confidence checks
-------------------------
+1. Confidence-checking FP32
```

Comment:
"Sanity check" is a being phased out as a non-inclusive term in the industry: https://inclusivenaming.org/
This should be changed in graphics as well.

Reply:
There's a new item in the backlog to link steps in the user guide to techniques in the feature guide. We should do that here as well.